README.md

```diff
@@ -284,6 +284,7 @@ No modules.
 | <a name="input_agents_type"></a> [agents\_type](#input\_agents\_type)                                                                                                         | (Optional) The type of Node Pool which should be created. Possible values are AvailabilitySet and VirtualMachineScaleSets. Defaults to VirtualMachineScaleSets.                                                                                                                                                                  | `string`                                                                                                                                                                                                                   | `"VirtualMachineScaleSets"` |    no    |
 | <a name="input_api_server_authorized_ip_ranges"></a> [api\_server\_authorized\_ip\_ranges](#input\_api\_server\_authorized\_ip\_ranges)                                       | (Optional) The IP ranges to allow for incoming traffic to the server nodes.                                                                                                                                                                                                                                                      | `set(string)`                                                                                                                                                                                                              | `null`                      |    no    |
+| <a name="input_automatic_channel_upgrade"></a> [automatic\_channel\_upgrade](#input\_automatic\_channel\_upgrade)                                                             | (Optional) The upgrade channel for this Kubernetes Cluster. Possible values are  `patch`, `rapid`, `node-image` and `stable`. By default automatic-upgrades are turned off. See [the documentation](https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-cluste) for more information                                        | `string`                                                                                                                                                                                                                   | `null`                      |    no    |
```

Comment:
The link to the documentation is wrong: you are missing a `r`
https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-cluste

Reply:
Could you also document that using this variable requires that you cannot specify a patch version in `kubernetes_version` ?

Reply:
Sure, added a sentence about this and fixed the typo.